controllers/userController.js

```diff
@@ -101,6 +102,41 @@ router.post('/:id/password', verifyToken, (req, res) => {
 	});
 });
 
+// Changes a don's information (only accessible by dons)
+router.post('/:id/doninfo', verifyToken, (req, res) => {
+	if(req.user.role&1 != 1) {
+		console.error("User "+ req.user._id +"is requesting don info when user is not a don");
+		//requestor is not a don
+		res.status(403);
+	}
+	User.findOne({_id: req.params.id}).then(user => {
```

Comment:
We should probably ensure the user making the request is only editing themselves (the `req.params.id !== req.user._id` check on a bunch of other user routes). Currently any don could change any other don's info (this would also remove the need for the check immediately below this)

Reply:
This is intentional. This way if a don forgets to change their status, another don can update it for them. Since they are the "responsible" ones, I don't see this as an issue. The front end only enables editing of other dons being on/off and late supper and not location or clockout, but I don't expect dons to be hacking other don's locations and clockout times outside of the app.

Reply:
:+1: